package-tests.cake

```diff
@@ -293,19 +293,20 @@ public class ZipPackageTester : NetFXPackageTester
     public ZipPackageTester(ICakeContext context, string packageVersion)
         : base(context, packageVersion)
     {
-        // Add tests specific to the zip package
-        PackageTests.Add(new PackageTest(
-            "Run project with both copies of mock-assembly",
-            $"../../NetFXTests.nunit --config={_config}",
-            new ExpectedResult("Failed")
-            {
-                Total = 2 * 37,
-                Passed = 2 * 23,
-                Failed = 2 * 5,
-                Warnings = 0,
-                Inconclusive = 2 * 1,
-                Skipped = 2 * 7
-            }));
+        // TODO: Get extensions working - need new releases due to interface change
```

Comment:
I'll be doing at least the NUnit Project as a new release while we work on this. Don't want to do too much until the API is established.

Reply:
Actually, it may only be the NUnit Project loader that has a problem... it uses the old string constructor, which now looks like a params string[] constructor!